TunnelKit/Sources/Protocols/OpenVPN/AppExtension/OpenVPNTunnelProvider+Configuration.swift

```diff
@@ -181,6 +181,8 @@ extension OpenVPNTunnelProvider {
             
             static let httpsProxy = "HTTPSProxy"
             
+            static let proxyAutoConfURL = "proxyAutoConfURL"
```

Comment:
Please replace this declaration with:

`static let proxyAutoConfigurationURL = "ProxyAutoConfigurationURL"`

which is more in accordance to (unsaid) key name conventions.

Needless to say, rename `proxyAutoConfigURL` to `proxyAutoConfigurationURL` everywhere else.